backend/protos/buf.gen.rust.yaml

```diff
@@ -0,0 +1,6 @@
+version: v1
```

Comment:
Nice! I think you can just call this buf.gen.yaml, I don't think there's anything that would automatically pick it up.

Reply:
it's currently in the same directory as the `buf.gen.yaml` we use for other protos- i could move it to the sqlc-gen-ftl directory but i figured we may use rust/rust protos for other things